src/com/google/javascript/jscomp/ProductionCoverageInstrumentationCallback.java

```diff
@@ -52,6 +53,8 @@
   private final AbstractCompiler compiler;
   private final ParameterMapping parameterMapping;
 
+  private static final String GLOBAL_FUNCTION_NAME = "<Anonymous>";
```

Comment:
ANONYMOUS_FUNCTION_NAME? not sure why it's called global

Reply:
I just global since this is also used when there is code in global scope that isn't a function. But changed to use Anonymous instead.